src/set_theory/game/pgame.lean

```diff
@@ -148,6 +148,32 @@ def move_right : Π (g : pgame), right_moves g → pgame
 @[simp] lemma right_moves_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).right_moves = xr := rfl
 @[simp] lemma move_right_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).move_right = xR := rfl
 
+/-- Conway induction on games. -/
+@[elab_as_eliminator] def move_rec_on {C : pgame → Sort*} (x : pgame)
+  (IH : ∀ (y : pgame), (∀ i, C (y.move_left i)) → (∀ j, C (y.move_right j)) → C y) : C x :=
+x.rec_on $ λ yl yr yL yR, IH (mk yl yr yL yR)
+
+/-- `is_option x y` means that `x` is either a left or right option for `y`. -/
+@[mk_iff] inductive is_option : pgame → pgame → Prop
+| move_left {x : pgame} (i) : is_option (x.move_left i) x
+| move_right {x : pgame} (i) : is_option (x.move_right i) x
+
+theorem is_option.mk_left {xl xr : Type u} {xL : xl → pgame} {xR : xr → pgame} {i : xl} :
+  (xL i).is_option (mk xl xr xL xR) :=
+@is_option.move_left (mk _ _ _ _) i
+
+theorem is_option.mk_right {xl xr : Type u} {xL : xl → pgame} {xR : xr → pgame} {i : xr} :
```

Comment:
I think these should be explicit (same above)
```suggestion
theorem is_option.mk_right {xl xr : Type u} (xL : xl → pgame) (xR : xr → pgame) (i : xr) :
```

Reply:
Oh, I thought you were talking about the `move_left` lemmas. Yeah, I agree.

Reply:
I was, this comment is independent